target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -564,6 +564,17 @@ define Device/tplink_tl-wpa8630p-v2-int
 endef
 TARGET_DEVICES += tplink_tl-wpa8630p-v2-int
 
+define Device/tplink_tl-wpa8730-v1
+  $(Device/tplink-8mlzma)
+  SOC := qca9558
+  DEVICE_MODEL := TL-WPA8730
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct
+  TPLINK_HWID := 0x87300001
+  IMAGE_SIZE := 8064k
```

Comment:
I don't think IMAGE_SIZE is used for anything for this device. I'd drop it.

Reply:
You're correct, the variable is not used for anything, because v1 doesn't include `check-size` for the images. `IMAGE_SIZE` is defined for `Device/tplink-8mlzma` from which we inherit and it doesn't match. Does it make sense to add `check-size` to the v1 targets?

For now I've removed it, because it's missing from the other images.